pkgpanda/__init__.py

```diff
@@ -532,6 +563,7 @@ def __init__(
         self.__manage_users = manage_users
         self.__add_users = add_users
         self.__manage_state_dir = manage_state_dir
+        self.__sysctl_config = sysctl_config if sysctl_config is not None else dcos_sysctl_config
```

Comment:
There is a subtle difference between these two expressions.

    >>> x = False or True
    >>> x
    True
    >>> x = "" or "/path/"
    >>> x
    '/path/'

For the practical purposes with respect to this file, the review comment is alright.